### PR TITLE
[14.0][FIX] rma_account: get correct account when a bill refund is created and reference to a supplier rma is made

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
@JuanyDForgeflow Can you review and add some test cases?

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]

File: rma_account/views/account_move_view.xml [hunk 1]
```diff
@@ -55,10 +50,10 @@
                  <field
                     name="add_rma_line_id"
                     context="{'rma': True}"
-                    domain="[('type', '=', 'customer'),('partner_id', '=', partner_id)]"
+                    domain="[('type', '=', 'supplier'),('partner_id', '=', partner_id)]"
                     attrs="{'readonly': [('state','not in',['draft'])],
                          'invisible': ['|', ('state', '=', 'paid'),
```
Comment (JuanyDForgeflow) [inline rma_account/views/account_move_view.xml:55]: `attrs="{'readonly': [('payment_state','not in',['draft'])],`

Comment (JuanyDForgeflow) [inline rma_account/views/account_move_view.xml]:
```suggestion
                         'invisible': ['|', ('payment_state', '=', 'paid'),
```

**Comment by JordiBForgeFlow** [comment]
@JuanyDForgeflow can you forward por to 15.0?

**Comment by JuanyDForgeflow** [comment]
@JordiBForgeFlow it's already done here https://github.com/ForgeFlow/stock-rma/pull/304

**Pull request merged**